Toolset/palettes/standalone settings/revstandalonesettingsiosbehavior.livecodescript

```diff
@@ -379,12 +389,17 @@ function getProfiles
 end getProfiles
 
 on updateIosSettingState
-   set the enabled of group "iOS Settings" of me to (the hilite of button "Build for iOS" of me)
-   set the visible of button "iOSChecked" to (the hilite of button "Build for iOS" of me)
```

Comment:
Those lines are needed since they update the "green tick" on the iOS icon at the top, when checking/unchecking the "Build for iOS checkbox":
```
set the visible of button "iOSChecked" to (the hilite of button "Build for iOS" of me)
set the opaque of button "iOSChecked" to false
```